errors/invalid-next-config.md

```diff
@@ -0,0 +1,13 @@
+# Invalid next.config.js
+
+#### Why This Error Occurred
+
+In your `next.config.js` file you passed invalid options that either are the incorrect type or an unknown field.
+
+#### Possible Ways to Fix It
+
+Fixing the listed config errors will remove this warning. You can also leverage the `NextConfig` type by importing from `next` to help ensure your config is correct.
```

Comment:
Let's give an example here?

```js
/**
 * @type {import('next').NextConfig}
 */
const nextConfig = {
  /* config options here */
}

module.exports = nextConfig
```

Reply:
Updated in https://github.com/vercel/next.js/pull/38498/commits/e8d4a8353c4b22b09840dac73f07ae7f5e860514